src/dlgprefreplaygaindlg.ui

```diff
@@ -148,15 +148,21 @@
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
+        <property name="toolTip">
+         <string>When ReplayGain is enabled, adjust tracks lacking ReplayGain information by this amount.</string>
+        </property>
         <property name="text">
-         <string>Inital boost without ReplayGain</string>
+         <string>Initial boost without ReplayGain</string>
```

Comment:
With this wording, it's not clear whether or not this boost is applied when ReplayGain (the feature) is off, or when ReplayGain data is not present.  The tooltip does explain this, but for users with tooltips off this will be confusing.  Maybe just "Initial boost without ReplayGain data"?